test/functional/wallet_listtransactions.py

```diff
@@ -91,18 +91,19 @@ def run_test(self):
                             {"category": "receive", "amount": Decimal("0.44")},
                             {"txid": txid})
 
-        pubkey = self.nodes[1].getaddressinfo(self.nodes[1].getnewaddress())['pubkey']
-        multisig = self.nodes[1].createmultisig(1, [pubkey])
-        self.nodes[0].importaddress(multisig["redeemScript"], "watchonly", False, True)
-        txid = self.nodes[1].sendtoaddress(multisig["address"], 0.1)
-        self.nodes[1].generate(1)
-        self.sync_all()
-        assert_equal(len(self.nodes[0].listtransactions(label="watchonly", include_watchonly=True)), 1)
-        assert_equal(len(self.nodes[0].listtransactions(dummy="watchonly", include_watchonly=True)), 1)
-        assert len(self.nodes[0].listtransactions(label="watchonly", count=100, include_watchonly=False)) == 0
-        assert_array_result(self.nodes[0].listtransactions(label="watchonly", count=100, include_watchonly=True),
-                            {"category": "receive", "amount": Decimal("0.1")},
-                            {"txid": txid, "label": "watchonly"})
+        if not self.options.descriptors:
```

Comment:
It's not necessary because it is testing the `include_watchonly` parameter in `listtransactions` which is not present in descriptor wallets.

Reply:
Added a comment. It is not necessary to test these for descriptor wallets, that's why they are disabled in the first place. Otherwise there would be a TODO.